torch/onnx/symbolic_opset12.py

```diff
@@ -13,3 +13,61 @@
 def einsum(g, equation, tensor_list):
     tensors = sym_help._unpack_list(tensor_list)
     return g.op("Einsum", *tensors, equation_s=equation)
+
+def nll_loss(g, self, target, weight, reduction, ignore_index):
+    # none reduction : onnx::Constant[value={0}]
+    # mean reduction : onnx::Constant[value={1}]
+    # sum reduction : onnx::Constant[value={2}]
+    reduction = sym_help._maybe_get_const(reduction, 'i')
+    reduction_vals = ['none', 'mean', 'sum']
+    reduction = reduction_vals[reduction]
+
+    # when ignore_index is not specified, ignore_index == onnx::Constant[value={-100}]
+    if sym_help._maybe_get_const(ignore_index, 'i') != -100:
```

Comment:
If `ignore_index` is specified, e.g. `ignore_index == onnx::Constant[value={1}]`, this condition would evaluate to True. I don't think this is intended as in these cases `ignore_index` is ignored.

Reply:
I refactored the code and missed to flip this. thanks!